src/backend/legendary/library.ts

```diff
@@ -247,42 +247,7 @@ export class LegendaryLibrary {
    * @returns App names of updateable games.
    */
   public async listUpdateableGames(): Promise<string[]> {
-    const isLoggedIn = LegendaryUser.isLoggedIn()
-
-    if (!isLoggedIn || !isOnline()) {
-      return []
-    }
-    const epicOffline = await isEpicServiceOffline()
-    if (epicOffline) {
-      logWarning(
-        'Epic servers are offline, cannot check for game updates',
-        LogPrefix.Backend
-      )
-      return []
-    }
-
-    const abortID = 'legendary-check-updates'
-    const res = await runLegendaryCommand(
-      ['list', '--third-party'],
-      createAbortController(abortID),
-      {
-        logMessagePrefix: 'Checking for game updates'
-      }
-    )
-
-    deleteAbortController(abortID)
-
-    if (res.abort) {
-      return []
-    }
-
-    if (res.error) {
-      logError(
-        ['Failed to check for game updates:', res.error],
-        LogPrefix.Legendary
-      )
-      return []
-    }
+    this.getGames(true)
```

Comment:
we can reuse `this.getGames` that does all the online checks and legendary call and also updates the list of games